project/settings.py

```diff
@@ -53,18 +55,25 @@ def get(variable):
     'apps.logging',
     'apps.user',
     'apps.mail',
+    # WEBSOCKETS - The necessary apps
     'apps.chatter',
     'channels'
 
 ]
 
+# WEBSOCKETS - RedisChannelLayer should be used in production - But will break in tests.
+# In development use InMemoryChannelLayer
 CHANNEL_LAYERS = {
     'default': {
         'BACKEND': 'channels_redis.core.RedisChannelLayer',
         'CONFIG': {
-            "hosts": [('127.0.0.1', 6379)],
-        },
-    },
+            "hosts": [(REDIS_HOST, REDIS_PORT)],
+        }
+     }
+} if ENV == PRODUCTION else {
```

Comment:
Haha oh man ternary looks crazy here. It works... but not sure I feel about it. But doesn't need to be changed.

Reply:
For reference, I think following this guideline for ternary expressions makes sense: https://google.github.io/styleguide/pyguide.html#211-conditional-expressions

This instance feels like and odd-duck, which is something I think Python likes to avoid :)